widget_driver_generator/lib/src/code_providers/driver_provider_code_provider.dart

```diff
@@ -137,6 +142,25 @@ class $_providerClassName extends WidgetDriverProvider<$_driverClassName> {
   $_driverClassName buildTestDriver() {
     return $_testDriverClassName();
   }
+
+  @override
+  void updateProvidedProperties($_driverClassName driver) {
+    /* 
```

Comment:
We should not use block comments according to `effective dart` ( dart guidelines by the dart team)
https://dart.dev/guides/language/effective-dart/documentation#dont-use-block-comments-for-documentation